src/test_lib_json/main.cpp

```diff
@@ -1757,6 +1757,27 @@ JSONTEST_FIXTURE(ValueTest, precision) {
     expected = "0.25634569487374054";
     result = Json::writeString(b, v);
     JSONTEST_ASSERT_STRING_EQUAL(expected, result);
+
+    b.settings_["precision"] = 5;
+    b.settings_["precisionType"] = "decimal";
```

Comment:
Maybe `floatPrecision` and `floatFormat`?

Reply:
* http://open-source-parsers.github.io/jsoncpp-docs/doxygen/class_json_1_1_stream_writer_builder.html

Oh, you're right. I forgot about that.

Yes, we need to keep the word "precision". I guess "precisionType" is fine.